### PR TITLE
Feature/splunk config

### PR DESCRIPTION
**Pull request by martinvisser**

### CODE REVIEW

File: git-plugin/src/main/kotlin/io/rabobank/ret/git/plugin/command/PullRequestOpenCommand.kt [hunk 1]
```diff
@@ -17,23 +18,26 @@ import picocli.CommandLine.ScopeType
...
+    // This is just needed for autocompletion.
+    // It allows to autocomplete PRs based on repository (ret pr open -r=rt <TAB>)
     @Option(
         names = ["--repository", "-r"],
         description = ["Filter on repository"],
         scope = ScopeType.INHERIT,
         completionCandidates = RepositoryFlagCompletionCandidates::class,
     )
-    var filterRepository: String? =
-        null // This is just needed for autocompletion. It allows to autocomplete PRs based on repository (ret pr open -r=rt <TAB>)
```
Comment (GuusdeWit) [inline git-plugin/src/main/kotlin/io/rabobank/ret/git/plugin/command/PullRequestOpenCommand.kt:36]: I'm not sure which part was just for autocompletion. It's just a parameter, Was it the completionCandidates perhaps? We might as well remove the comment completely

**Comment by martinvisser** [comment]
Needs to wait for https://github.com/rabobank/ret-engineering-tools/pull/73 as this will release `0.2.0`.

File: splunk-plugin/src/test/kotlin/io/rabobank/ret/splunk/plugin/AutoCompleteCommandTest.kt [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    fun `should return all indexes if no word given`() {
+        val exitCode = commandLine.execute("indexes")
+        assertThat(exitCode).isEqualTo(0)
+
+        verify(outputHandler).listIndexes(indexes)
+    }
+
+    @ParameterizedTest
+    @MethodSource("matchIndexes")
+    fun `should return only matched indexes`(word: String, expectedIndexes: List<String>) {
```
Comment (GuusdeWit) [inline splunk-plugin/src/test/kotlin/io/rabobank/ret/splunk/plugin/AutoCompleteCommandTest.kt:41]: Should we add a similar test for projects?

File: git-plugin/pom.xml [hunk 3]
```diff
@@ -47,21 +48,36 @@
...
-            <artifactId>kotlin-stdlib</artifactId>
-        </dependency>
     </dependencies>
+
     <build>
         <finalName>git-plugin</finalName>
         <sourceDirectory>src/main/kotlin</sourceDirectory>
         <testSourceDirectory>src/test/kotlin</testSourceDirectory>
+
+        <!--        <resources>-->
```
Comment (GuusdeWit) [inline git-plugin/pom.xml:58]: Can we remove this?

**Review by aaukhatov — APPROVED** [review]

**Review by GuusdeWit — APPROVED** [review]

**Pull request merged**